doc/user_guide/user_guide.md

```diff
@@ -188,88 +191,31 @@ Each algorithm should extend the `UDFQueryHandler` abstract class and then imple
-from exasol_advanced_analytics_framework.query_handler.result import Result, Continue, Finish
-from exasol_advanced_analytics_framework.query_handler.query.select_query import SelectQuery, SelectQueryWithColumnDefinition
-from exasol_data_science_utils_python.schema.column import Column
-from exasol_data_science_utils_python.schema.column_name import ColumnName
-from exasol_data_science_utils_python.schema.column_type import ColumnType
-
-
-class ExampleQueryHandler(UDFQueryHandler):
-    def __init__(self, parameter: str, query_handler_context: QueryHandlerContext):
-        super().__init__(parameter, query_handler_context)
-        self.parameter = parameter
-        self.query_handler_context = query_handler_context
-
-    def start(self) -> Union[Continue, Finish[str]]:
-        query_list = [
-          SelectQuery("SELECT 1 FROM DUAL"),
-          SelectQuery("SELECT 2 FROM DUAL")]
-        query_handler_return_query = SelectQueryWithColumnDefinition(
-            query_string="SELECT 5 AS 'return_column' FROM DUAL",
-            output_columns=[
-              Column(ColumnName("return_column"), ColumnType("INTEGER"))])
-
-        return Continue(
-            query_list=query_list,
-            input_query=query_handler_return_query)
+The example dynamically creates a python module `xyz` and adds classes `ExampleQueryHandler` and `ExampleQueryHandlerFactory` to it.
 
-    def handle_query_result(self, query_result: QueryResult) -> Union[Continue, Finish[str]]:
-        return_value = query_result.return_column
-        result = 2 ** return_value
-        return Finish(result=result)
+In order to execute the example successfully you need to
+1. [Create a BucketFS connection](#bucketfs-connection)
+2. Activate the AAF's SLC
+3. Make sure the database schemas used in the example exist.
 
-import builtins
-builtins.ExampleQueryHandler=ExampleQueryHandler # required for pickle
+The example assumes
+* the name for the BucketFS Connection `<CONNECTION_NAME>` to be `BFS_CON`
+* the name for the AAF database schema `<AAF_DB_SCHEMA` to be `AAF_DB_SCHEMA`, see [Additional Scripts](#additional-scripts)
 
-class ExampleQueryHandlerFactory:
-      def create(self, parameter: str, query_handler_context: QueryHandlerContext):
-          return builtins.ExampleQueryHandler(parameter, query_handler_context)
+The following SQL statements activate the AAF's SLC and create the required database schemas unless they already exist:
 
-builtins.ExampleQueryHandlerFactory=ExampleQueryHandlerFactory
-
-from exasol_advanced_analytics_framework.udf_framework.query_handler_runner_udf \
-    import QueryHandlerRunnerUDF
+```shell
+ALTER SESSION SET SCRIPT_LANGUAGES='R=builtin_r JAVA=builtin_java PYTHON3=builtin_python3 PYTHON3_AAF=localzmq+protobuf:///bfsdefault/default/temp/exasol_advanced_analytics_framework_container_release?lang=python#/buckets/bfsdefault/default/temp/exasol_advanced_analytics_framework_container_release/exaudf/exaudfclient_py3';
```

Comment:
I applied your proposal, @tkilias.
How about the assumptions? 
Maybe we first try to identify all assumptions?

Thinking about it: maybe we can omit the ALTER SESSION statement, as all this is already done by the AAF deploy command?

Reply:
lets remove it

Reply:
:+1: